packages/core/src/types/config.ts

```diff
@@ -152,7 +152,7 @@ export type DragConfig = Omit<CoordinatesConfig<'drag'>, 'axisThreshold' | 'boun
    * Limits the gesture `offset` to the specified bounds. Can be a ref or a dom
    * node.
    */
-  bounds?: DragBounds | ((state: State['drag']) => DragBounds)
+  bounds?: DragBounds | ((state: StateTypes['drag']) => DragBounds)
```

Comment:
cared for
<img width="612" alt="image" src="https://user-images.githubusercontent.com/29725082/187327363-52350478-8a08-4300-8662-3d68488bd474.png">